pom.xml

```diff
@@ -529,6 +503,22 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-javadoc-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>attach-javadocs</id>
+                        <goals>
+                            <goal>jar</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <release>11</release>
```

Comment:
Looks like the [default value](https://maven.apache.org/plugins/maven-javadoc-plugin/javadoc-mojo.html#release) for `release` is `maven.compiler.release`, which is already **[11](https://github.com/spotify/github-java-client/blob/master/pom.xml#L422)** in this case.

Reply:
Yea that's what I assumed too, but it doesn't work anymore when I remove that line:
```
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  25.451 s
[INFO] Finished at: 2022-04-14T15:49:10+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:3.3.2:jar (attach-javadocs) on project github-client: MavenReportException: Error while generating Javadoc:
[ERROR] Exit code: 1 - /Users/nikod/dev/github-java-client/src/main/java/com/spotify/github/v3/repos/BranchDeserializer.java:56: warning: as of release 10, 'var' is a restricted local variable type and cannot be used for type declarations or as the element type of an array
[ERROR]     var immutableBranchBuilder = ImmutableBranch.builder();
[ERROR]         ^
[ERROR] /Users/nikod/dev/github-java-client/target/generated-sources/annotations/com/spotify/github/ImmutableLinks.java:27: error: cannot find symbol
[ERROR] @javax.annotation.processing.Generated("org.immutables.processor.ProxyProcessor")
[ERROR]                             ^
[ERROR]   symbol:   class Generated
[ERROR]   location: package javax.annotation.processing
[ERROR] /Users/nikod/dev/github-java-client/target/generated-sources/annotations/com/spotify/github/ImmutableHref.java:28: error: cannot find symbol
[ERROR] @javax.annotation.processing.Generated("org.immutables.processor.ProxyProcessor")
[ERROR]                             ^
```

I have no idea why and honestly don't want to spend too much time on this. If you know a better way please let me know :)

Reply:
If you have already tried that, then it is okay to have the version explicitly mentioned.